core/common/src/test/java/alluxio/ConfigurationTestUtils.java

```diff
@@ -125,7 +126,10 @@ public static InstancedConfiguration defaults() {
     conf.put(PropertyKey.WORKER_NETWORK_SHUTDOWN_TIMEOUT, "0ms");
 
     conf.put(PropertyKey.Template.WORKER_TIERED_STORE_LEVEL_ALIAS.format(0), "MEM");
-    conf.put(PropertyKey.USER_RPC_RETRY_MAX_DURATION, "1s");
+    if (journalType.equals("UFS")) {
+      // Raft journal system need longer to start, job worker connect should wait for longer time
+      conf.put(PropertyKey.USER_RPC_RETRY_MAX_DURATION, "1s");
```

Comment:
can we make the raft journal start faster by tuning `MASTER_EMBEDDED_JOURNAL_ELECTION_TIMEOUT` and `MASTER_EMBEDDED_JOURNAL_HEARTBEAT_INTERVAL`? We want the test to run as fast as possible without introducing flakiness.

Reply:
Added these properties to make the raft journal start faster. 

```
      conf.put(PropertyKey.USER_RPC_RETRY_MAX_DURATION, "30s");
      conf.put(PropertyKey.MASTER_EMBEDDED_JOURNAL_ELECTION_TIMEOUT, "1s");
      conf.put(PropertyKey.MASTER_EMBEDDED_JOURNAL_HEARTBEAT_INTERVAL, "100ms");
```

Reply:
it still takes 30 seconds for the raft journal to start?

Reply:
Changed to `3s` which is sufficient in my local test. Will change it to larger values if jenkins tests says to.